src/lib/rnpsdk.h

```diff
@@ -77,4 +78,8 @@ int64_t rnp_filemtime(const char *path);
 
 const char *rnp_filename(const char *path);
 
+int grabdate(const char *s, int64_t *t);
```

Comment:
Ok, commented in the code. Actually it was old function just moved from the wrong place. Now it is in the CLI.